src/main/java/io/Printer.java

```diff
@@ -0,0 +1,43 @@
+package io;
+
+import domain.Car;
+
+import java.util.List;
+
+public class Printer {
+    Message message = new Message();
+
+    private static String DEFAULT_SYMBOL = "-";
+
+    public void printProgressWithSymbol(String name, int position) {
+        String result = "";
+        for(int i = 0;i < position;i++)
+            result += DEFAULT_SYMBOL;
```

Comment:
핵데이 컨벤션 8.3, 8.4를 참고해주세요! [[참조]](https://naver.github.io/hackday-conventions-java/#space-between-keyword-parentheses)

```suggestion
        for (int i = 0; i < position; i++) {
            result += DEFAULT_SYMBOL;
        }
```

또한 Car 객체의 상태를 보여주는 문자열의 생성도 다른 객체에게 위임할 수 있지 않을까요?
객체의 자율성을 생각한다면 Car 객체 자신이 자신의 상태를 보여주는 문자열을 만들 수 있을 것 같습니다!